src/services/summarize-state/index.js

```diff
@@ -0,0 +1,80 @@
+const _ = require('lodash');
+const hooks = require('./hooks');
+
+const pinStates = require('../../constants/pin-states');
+
+// Models
+const Department = require('../department/department-model');
+const Organization = require('../organization/organization-model');
+const Pin = require('../pin/pin-model');
+
+class Service {
+  constructor(options) {
+    this.options = options || {};
+  }
+
+  find(params) {
+    const startDate = params.query.start_date;
+    const endDate = params.query.end_date;
+    const organizationName = params.query.organization;
+    if (!startDate || !endDate) {
+      return Promise.reject(new Error('Please specified both start_date and end_date'));
+    }
+    // Find organization id
+    return Organization.findOne({ name: organizationName })
```

Comment:
Finding organization might not be necessary because we already remove multiple organizations out. So, we can simply use `Pin.find(query)` here.